app/views/idv/by_mail/letter_enqueued/show.html.erb

```diff
@@ -19,21 +19,33 @@
 
 <%= render PageHeadingComponent.new.with_content(t('idv.titles.come_back_later')) %>
 
-<p><%= t('idv.gpo.will_send_to') %></p>
+<p><%= t('idv.gpo.will_send_to', app_name: APP_NAME) %></p>
 
 <div class="padding-y-1">
   <% @presenter.address_lines.each do |address_line| %>
     <p class="margin-bottom-0"><%= address_line %></p>
   <% end %>
 </div>
 
-<p class="padding-top-3 padding-bottom-3">
-  <%= t('idv.messages.come_back_later_html') %>
-</p>
+<% if @presenter.has_sp? %>
+  <ul class="usa-list">
+    <% t(
+         'idv.messages.come_back_later_html',
+         sp_link: link_to(@presenter.sp_name, @presenter.button_destination),
+       ).each do |bullet| %>
+      <li><%= bullet %></li>
+    <% end %>
+  </ul>
+<% else %>
+  <p class="margin-top-2">
+    <%= t('idv.messages.come_back_later_no_sp_html') %>
+  </p>
+<% end %>
```

Comment:
And, context here: if there is no SP, we don't want to show the second bullet point telling them to contact `nil`, so we omit it. But a one-item bulleted list is silly, so we go for a paragraph tag instead.

Reply:
This is a good catch! I will update #11351 to do the same.